x-pack/plugins/actions/server/builtin_action_types/case/utils.ts

```diff
@@ -0,0 +1,253 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { curry, flow } from 'lodash';
+import { schema } from '@kbn/config-schema';
+import { AxiosInstance, Method, AxiosResponse } from 'axios';
+
+import { ActionTypeExecutorOptions, ActionTypeExecutorResult, ActionType } from '../../types';
+
+import { ExecutorParamsSchema } from './schema';
+
+import {
+  CreateConnectorArgs,
+  ConnectorPublicConfigurationType,
+  CreateActionTypeArgs,
+  ExecutorParams,
+  MapRecord,
+  AnyParams,
+  CreateConnectorBasicArgs,
+  PrepareFieldsForTransformArgs,
+  PipedField,
+  TransformFieldsArgs,
+  Comment,
+  ExecutorActionParams,
+} from './types';
+
+import * as transformers from './transformers';
```

Comment:
we should do it like that to simplify types `import { transformers } from './transformers';` and in transformer file do that
```
export const transformers: Record<string, (args: TransformerArgs) => TransformerArgs> = {
  informationCreated: ({ value, date, user, ...rest }: TransformerArgs): TransformerArgs => ({
    value: `${value} ${i18n.FIELD_INFORMATION('create', date, user)}`,
    ...rest,
  }),
  informationUpdated: ({ value, date, user, ...rest }: TransformerArgs): TransformerArgs => ({
    value: `${value} ${i18n.FIELD_INFORMATION('update', date, user)}`,
    ...rest,
  }),
  informationAdded: ({ value, date, user, ...rest }: TransformerArgs): TransformerArgs => ({
    value: `${value} ${i18n.FIELD_INFORMATION('add', date, user)}`,
    ...rest,
  }),
  append: ({ value, previousValue, ...rest }: TransformerArgs): TransformerArgs => ({
    value: previousValue ? `${previousValue} \r\n${value}` : `${value}`,
    ...rest,
  }),
};
```

Reply:
Yes, that's much better. It will help us to avoid `const t = { ...transformers } as { [index: string]: Function };`